### ISSUE TITLE
Babble error in 2.2.0.beta8

### ISSUE THREAD

**Issue by azawawi** [gdpelican/babble#275]
First of all, Thanks for your awesome plugin. :+1: 

While upgrading from 2.2.0.beta7 to [2.2.0.beta8](https://meta.discourse.org/t/discourse-2-2-0-beta8-release-notes/105915). I found that babble stopped working. The following error is in the logs. It seems related to Discourse's upgrade from Ember 2.x to Ember 3.5. Direct chat message also keeps having the loading animation and does not open.

```
TypeError: Cannot set property postStream of [object Object] which has only a getter
    at d (https://localhost/assets/plugin-third-party-faa3d285b7e48db345236a7d099b22070086c819b5057d99f2d4719ede34d0d0.js:2:5080)
    at i.buildTopic (https://localhost/assets/plugin-third-party-faa3d285b7e48db345236a7d099b22070086c819b5057d99f2d4719ede34d0d0.js:2:12160)
    at i.setupTopicListener (https://localhost/assets/plugin-third-party-faa3d285b7e48db345236a7d099b22070086c819b5057d99f2d4719ede34d0d0.js:2:10986)
    at https://localhost/assets/plugin-third-party-faa3d285b7e48db345236a7d099b22070086c819b5057d99f2d4719ede34d0d0.js:2:10320
    at Function.Nt (https://localhost/assets/vendor-3eae3eec8fba033cb1b5af6c103a5b8781caa69cf97fe7773ba89ba68980b263.js:3:27206)
    at https://localhost/assets/plugin-third-party-faa3d285b7e48db345236a7d099b22070086c819b5057d99f2d4719ede34d0d0.js:2:10292
    at h (https://localhost/assets/ember_jquery-cf9339810550f9c92505dfbb37362c58a4a8a83bcee2d99174547b01c06ed7d3.js:18:1869)
    at C (https://localhost/assets/ember_jquery-cf9339810550f9c92505dfbb37362c58a4a8a83bcee2d99174547b01c06ed7d3.js:18:3255)
    at E (https://localhost/assets/ember_jquery-cf9339810550f9c92505dfbb37362c58a4a8a83bcee2d99174547b01c06ed7d3.js:18:3148)
    at e.invoke (https://localhost/assets/ember_jquery-cf9339810550f9c92505dfbb37362c58a4a8a83bcee2d99174547b01c06ed7d3.js:9:11634)
```

**Comment by gdpelican** [comment]
I've just pushed a fix for this in 4.0.10, could you try rebuilding again and seeing if that fixes the problem?

**Comment by azawawi** [comment]
@gdpelican Works great. Thanks a lot :+1:
